packages/ui-extensions/src/surfaces/checkout/api/standard/standard.ts

```diff
@@ -78,6 +78,8 @@ export interface Extension<Target extends ExtensionTarget = ExtensionTarget> {
    * * [`collect_buyer_consent.sms_marketing`](https://shopify.dev/docs/api/checkout-ui-extensions/configuration#collect-buyer-consent): the extension can collect customer consent for SMS marketing.
    *
    * * [`collect_buyer_consent.customer_privacy`](https://shopify.dev/docs/api/checkout-ui-extensions/configuration#collect-buyer-consent): the extension can register customer consent decisions that will be honored on Shopify-managed services.
+   *
+   * * [`iframe.sources`](https://shopify.dev/docs/api/checkout-ui-extensions/configuration#iframe): the extension can embed an external URL in an iframe.
```

Comment:
```suggestion
   * * `iframe.sources`: the extension can embed an external URL in an iframe.
```
Nit: same note as above: the docs won't include the new capabilities with this update. @jffortier should we remove the link for now?

Reply:
Good catch. @hheyhhay can you commit this suggestion for your PR and we'll make sure it gets re-added in the next release?